cSpell.json

```diff
@@ -0,0 +1,108 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": [
+    "Alloc",
+    "analyse",
+    "analyser",
+    "Analysers",
+    "Autofac",
+    "bitness",
+    "corlib",
+    "Cygwin",
+    "Diagnoser",
+    "diagnosers",
+    "disassemblers",
+    "disassm",
+    "Jits",
+    "Jitting",
+    "LINQ",
+    "microbenchmarking",
+    "microbenchmarks",
+    "Mispredict",
+    "Mispredictions",
+    "msbuild",
+    "Multimodal",
+    "multimodality",
+    "netcoreapp",
+    "powerplans",
+    "Pseudocode",
+    "runtimes",
+    "Serilog",
+    "Tailcall",
+    "toolchains",
+    "unmanaged"
+  ],
+  "ignoreWords": [
+    "Akinshin",
+    "Andrey",
+    "Expecto",
+    "Jint",
+    "LoongArch64",
+    "macrobenchmark",
+    "MediatR",
+    "Nagórski's",
+    "Newtonsoft",
+    "NodaTime",
+    "Npgsql",
+    "Sitnik's",
+    "Wojciech",
+    "Avalonia",
+    "Gitter"
+  ],
+  "patterns": [
+    {
+      "name": "Markdown links",
+      "pattern": "\\((.*)\\)",
+      "description": ""
+    },
+    {
+      "name": "Markdown code blocks",
+      "pattern": "/^(\\s*`{3,}).*[\\s\\S]*?^\\1/gmx",
+      "description": "Taken from the cSpell example at https://cspell.org/configuration/patterns/#verbose-regular-expressions"
+    },
+    {
+      "name": "Inline code blocks",
+      "pattern": "\\`([^\\`\\r\\n]+?)\\`",
+      "description": "https://stackoverflow.com/questions/41274241/how-to-capture-inline-markdown-code-but-not-a-markdown-code-fence-with-regex"
+    },
+    {
+      "name": "Link contents",
+      "pattern": "\\<a(.*)\\>",
+      "description": ""
+    },
+    {
+      "name": "Snippet references",
+      "pattern": "-- snippet:(.*)",
+      "description": ""
+    },
+    {
+      "name": "Snippet references 2",
+      "pattern": "\\<\\[sample:(.*)",
+      "description": "another kind of snippet reference"
+    },
+    {
+      "name": "Multi-line code blocks",
+      "pattern": "/^\\s*```[\\s\\S]*?^\\s*```/gm"
+    },
+    {
+      "name": "HTML Tags",
+      "pattern": "<[^>]*>",
+      "description": "Reference: https://stackoverflow.com/questions/11229831/regular-expression-to-remove-html-tags-from-a-string"
+    }
+  ],
+  "ignoreRegExpList": [
+    "Markdown links",
+    "Markdown code blocks",
+    "Inline code blocks",
+    "Link contents",
+    "Snippet references",
+    "Snippet references 2",
+    "Multi-line code blocks",
+    "HTML Tags"
+  ],
+  "ignorePaths": [
```

Comment:
ℹ️ I ignore two globs as part of this PR:

* Changelog -- it usually contains many names, and spelling issues could arise as the result of direct copy/paste. More messy to leave in than to take out typically I find, so I ignore it. If you'd like me to un-ignore it and work through it, I can do that.
* Team page -- mostly due to the names. However, as above, happy to un-ignore and work through it if that's your preference.